contrib/cirrus/runner.sh

```diff
@@ -233,6 +233,24 @@ function _run_consistency() {
     SUGGESTION="run 'make generate-bindings' and commit all changes" ./hack/tree_status.sh
     make completions
     SUGGESTION="run 'make completions' and commit all changes" ./hack/tree_status.sh
+
+    if  [[ -z "$CIRRUS_TAG" ]] && \
+        req_env_vars CIRRUS_CHANGE_IN_REPO CIRRUS_PR DEST_BRANCH
+    then
+        msg "#####"
+        msg "Verifying no change adds new calls to io/ioutils."
+        local base diffs regex i_char
+        base=$(git merge-base $DEST_BRANCH $CIRRUS_CHANGE_IN_REPO)
+        diffs=$(git diff $base $CIRRUS_CHANGE_IN_REPO)
+        # Prevent this check from detecting itself (0x69 == "i")
+        regex=$(echo -e "^(\\+.+io/\x69outils)|(\\+.+\x69outils\\..+)")
+        if egrep -q "$regex"<<<"$diffs"; then
```

Comment:
Ugh, somehow [this is still matching line 246](https://cirrus-ci.com/task/5306032040706048) despite the obfuscation.

Reply:
No, it's matching the one two lines down.

I have strong concerns about this approach, but won't have time to fully review until later today.

Reply:
Oh, it's the messages.